packages/bpk-scrim-utils/src/withScrimmedPortal-test.tsx

```diff
@@ -105,4 +106,48 @@ describe('withScrimmedPortal', () => {
 
     expect(screen.getByText('Wrapped Component / portal is not ready yet / portal is now ready')).toBeInTheDocument();
   });
+
+  it('renders the wrapped component inside a portal correctly when runOnServer is false (default)', () => {
+    const DialogContent = () => <div>Dialog content</div>;
+    const ScrimmedComponent = withScrimmedPortal(DialogContent);
+
+    render(
+      <div id="pagewrap">
+        <div> Content hidden from AT</div>
+        <ScrimmedComponent
+          getApplicationElement={() => document.getElementById('pagewrap')}
+        />
+      </div>
+    );
+    expect(document.body).toMatchSnapshot();
+  });
+
+  it('renders the wrapped component inside a portal correctly when runOnServer is true', () => {
+    const DialogContent = () => <div>Dialog content</div>;
+    const ScrimmedComponent = withScrimmedPortal(DialogContent);
+
+    render(
+      <div id="pagewrap">
+        <div> Content hidden from AT</div>
+        <ScrimmedComponent
```

Comment:
where is the `runOnServer` prop?

Reply:
Oh... :/
I think I just tabbed on the suggestion from Copilot and missed that it didn't add the prop 😅